composer/blog/gcp-tech-blog/data-orchestration-with-composer/requirements.txt

```
@@ -0,0 +1,4 @@
+apache-airflow[google]==2.1.0
```

```suggestion
apache-airflow[google]==2.0.1
```

2.0.1 is the latest version offered in composer in public preview